frontend/src/scenes/scenes.ts

```diff
@@ -188,8 +189,10 @@ export const routes: Record<string, Scene> = {
     [urls.dashboard(':id')]: Scene.Dashboard,
     [urls.createAction()]: Scene.Action,
     [urls.action(':id')]: Scene.Action,
-    [urls.newInsight()]: Scene.InsightRouter,
-    [urls.insights()]: Scene.Insights,
+    [urls.newInsight()]: Scene.Insights,
+    [urls.insightEdit(':id')]: Scene.Insights,
+    [urls.insightView(':id')]: Scene.Insights,
+    [urls.savedInsights()]: Scene.SavedInsights,
```

Comment:
I did the first rename (`Scene.Insights` -> `Scene.Insight`), but I'm hesitant to do the second one. I'd like to, but to complete it means to rename `/scenes/saved-insights` and `SavedInsights.tsx` to `/scenes/insights`, and move the current `/scenes/insights` to `/scenes/insight`. That's because esbuild's chunking wants the Scene's name to [match](https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/sceneTypes.ts#L3) the exported class to be able to preload chunks.

I'd be up for that, but in a different PR to keep this manageable.